config/clients/go/config.overrides.json

```diff
@@ -64,12 +64,12 @@
       "templateType": "SupportingFiles"
     },
     "oauth2/token_test.go": {},
-    "utils/utils.mustache": {
-      "destinationFilename": "utils/utils.go",
+    "internal/utils/utils.mustache": {
+      "destinationFilename": "internal/utils/utils.go",
       "templateType": "SupportingFiles"
     },
-    "utils/utils_test.mustache": {
-      "destinationFilename": "utils/utils_test.go",
+    "internal/utils/utils_test.mustache": {
```

Comment:
This doesn't seem to match the generated SDK 🤔

Reply:
Did not push the changes :P